### PR TITLE
Readmes updates

### PR DESCRIPTION
**Pull request by agagancarczyk**
## Motivation
Updated all READMEs in order to bring more clarity to the project and sub-projects structures. 

## Verification Steps
N/A

## Checklist:

- [ ] Code has been tested locally by PR requester
- [ ] User-visible strings are using the react-i18next framework (useTranslation)
- [ ] Help has been implemented
- [ ] axe report has been run and resulting a11y issues have been resolved
- [ ] Unit tests have been created/updated

### CODE REVIEW

**Review by edewit — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,6 +2,33 @@
...
+    │   ├── admin-ui
+    │   └── keycloak-server
+    ├── keycloak-theme          # Maven build for the Keycloak theme
+    ├── libs                    # Keycloak libraries
+    ├── LICENSE
+    ├── MAINTAINERS.md
+    ├── README.md
+    ├── package.json
+    ├── tsconfig.eslint.json
+    └── tsconfig.json
```
Comment (jonkoops) [inline README.md:18]: I don't think we need to list all of these files, they have little relevance to the directory structure.
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -2,6 +2,33 @@
...
 This repository contains the UIs and related libraries of the Keycloak project.
 
+## Repository structure
+
+    ├── apps                    # UI projects
+    │   ├── account-ui
+    │   ├── admin-ui
+    │   └── keycloak-server
+    ├── keycloak-theme          # Maven build for the Keycloak theme
+    ├── libs                    # Keycloak libraries
```
Comment (jonkoops) [inline README.md:12]: It would be nice to have the libraries listed here as well, with a nice description.

File: README.md [hunk 3]
```diff
@@ -2,6 +2,33 @@
 
 This repository contains the UIs and related libraries of the Keycloak project.
 
+## Repository structure
+
+    ├── apps                    # UI projects
+    │   ├── account-ui
+    │   ├── admin-ui
+    │   └── keycloak-server
```
Comment (jonkoops) [inline README.md:10]: Can we add comments here as well with the description of each directory?

File: apps/account-ui/README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# Keycloak Account UI
+
+This project is the next generation of the Keycloak Account UI. It is written with React and [PatternFly 4](https://www.patternfly.org/v4/).
+
+## Repository structure
```
Comment (jonkoops) [inline apps/account-ui/README.md:5]: I think there is no need to describe the directory structure here, it follows the conventions of the Vite template. Perhaps a callout to the [Vite guide](https://vitejs.dev/guide/) here would be enough.

File: apps/admin-ui/README.md [hunk 5]
```diff
@@ -2,6 +2,41 @@
 
 This project is the next generation of the Keycloak Administration UI. It is written with React and [PatternFly 4](https://www.patternfly.org/v4/).
 
+## Repository structure
```
Comment (jonkoops) [inline apps/admin-ui/README.md:5]: Same here, I think callouts to the Vite and Cypress guides would be enough.

File: apps/keycloak-server/README.md [hunk 6]
```diff
@@ -2,6 +2,16 @@
 
 This app allows you to run a local development version of the Keycloak server.
 
+## Repository structure
```
Comment (jonkoops) [inline apps/keycloak-server/README.md:5]: I also think there is no need for the directory structure here.

File: apps/keycloak-server/README.md [hunk 7]
```diff
@@ -2,6 +2,16 @@
...
+
+    ├── scripts                    # Keycloak server config scripts
+    ├── README.md
+    ├── package.json
+
+### Keycloak server config scripts
```
Comment (jonkoops) [inline apps/keycloak-server/README.md:11]: This is not really relevant as these scripts are ran through NPM, and the rest is just implementation details.

**Review by jonkoops — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (agagancarczyk) [reply README.md:18]: Thanks, yeah, I was thinking the same but wanted to see what you would think of it.

**Comment by agagancarczyk** [comment]
hi @jonkoops could you please review it again? Let's finalize it. Thanks!

**Pull request merged**